openjdk/excludes/ProblemList_openjdk8.txt

```diff
@@ -361,6 +361,7 @@ com/sun/jdi/PrivateTransportTest.sh https://github.com/adoptium/aqa-tests/issues
 # jdk_util
 
 java/util/Arrays/TimSortStackSize2.java https://github.com/adoptium/aqa-tests/issues/1051 linux-s390x
+java/util/Currency/ValidateISO4217.java https://github.com/eclipse-openj9/openj9/issues/21539 generic-all
```

Comment:
I think it's better problemlist this testcase in [ProblemList_openjdk8-openj9.txt](https://github.com/adoptium/aqa-tests/blob/master/openjdk/excludes/ProblemList_openjdk8-openj9.txt), becase this test run passed by openjdk8, it only failed runed by openj9?

Reply:
No - as per the description `This has been seen on the Temurin builds too` (which are not J9)
Alhuogh since the upstream fix has gone in we probably don't need to exclude this any more. I'll re-check it.